bird_classify.py

```diff
@@ -69,17 +68,20 @@ def do_training(results,last_results,top_k):
     old_labels = [label[0] for label in last_results]
     shared_labels  = set(new_labels).intersection(old_labels)
     if len(shared_labels) < top_k:
-      print('Difference detected')
-      return True
+        print('Difference detected')
+    return True
```

Comment:
Shouldn't this return be in the if statement

Reply:
Yes, it needs to be in the if statement. Thanks!